install_scripts/templates/common/docker-install.sh

```diff
@@ -227,16 +227,6 @@ _installDocker() {
             yum install -y yum-utils
             logSuccess "Installed yum-utils"
         fi
-        getUrlCmd
```

Comment:
I think you can remove this entire block from line 220 as you do not need yum-utils if you are not going to use yum-config-manager.

One thing to note is that this assumes that appstream repo is enabled on rhel. I think it is best that the script fail if it is not but it may end in a support case and we need to instruct the customer to enable.

```
Installing:
 container-selinux                               noarch                               2:2.229.0-2.module+el8.10.0+21196+3f0abbca                                 rhui-rhel-8-for-x86_64-appstream-rhui-rpms                                71 k
```

I tested this on gcp with image rhel-8-v20240515 and it seemed to work.

Reply:
I also tested on azure since it is in the comment and that worked as well.

Reply:
Thank you @emosbaugh! I have removed the yum-utils install part